crates/storage/errors/src/lib.rs

```diff
@@ -9,7 +9,8 @@
 #![cfg_attr(docsrs, feature(doc_cfg, doc_auto_cfg))]
 #![cfg_attr(not(feature = "std"), no_std)]
 
-#[cfg(not(feature = "std"))]
+// #[cfg(not(feature = "std"))]
+// #[macro_use]
```

Comment:
This ended up not being issue because there aren't any tests specific to `reth-storage-errors` but would love someone else's thoughts on this. I'm kind of going for an no_std first approach and `std` being opt in instead of the default.

Reply:
removing it